cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -139,24 +145,24 @@ export class PassportAuth implements EndpointSecurity {
    *
    * @param passportApi - passport.js instance
    */
-  protected setup(passportApi: passport.Passport) {
-    const options = {
-      jwtFromRequest: ExtractJwt.fromAuthHeader(),
-      secretOrKey: 'secret'
-    };
-    passportApi.use(new Strategy(defaultStrategy(this.userRepo, this.userService)));
-    passportApi.use(new JwtStrategy(options, (jwtPayload, done) => {
-      const callback = (err?: Error, user?: any) => {
-        if (err) {
-          return done(err, false);
-        } else if (user) {
-          return done(null, user);
-        } else {
-          return done(null, false);
-        }
-      };
-      this.userRepo.getUserByLogin(jwtPayload.username, callback);
-    }));
+  protected setup(passportApi: passport.Passport, jwtOpts?: JwtOptions) {
+    if (jwtOpts) {
+      passportApi.use(new JwtStrategy(jwtOpts, (jwtPayload, done) => {
+        const callback = (err?: Error, user?: any) => {
+          if (err) {
+            return done(err, false);
+          } else if (user) {
+            return done(null, user);
+          } else {
+            return done(null, false);
+          }
+        };
+        this.userRepo.getUserByLogin(jwtPayload.username, callback);
+      }));
+    } else {
+      passportApi.use(new Strategy(defaultStrategy(this.userRepo, this.userService)));
```

Comment:
btw. - wit this change our `defaultStrategy` may not be as accurate as before. WDYT to rename this to WebStrategy to reflect the use case?